src/suites/cts/command_buffer/render/storeop_clear.spec.ts

```diff
@@ -0,0 +1,133 @@
+export const description = `
+renderPass store op clear`;
+
+import { TestGroup } from '../../../../framework/index.js';
+import GLSL from '../../../../tools/glsl.macro.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+g.test('drawn quad', async t => {
+  const sampledTexture = t.device.createTexture({
+    size: { width: 1, height: 1, depth: 1 },
+    format: 'r8unorm',
+    usage: GPUTextureUsage.SAMPLED | GPUTextureUsage.COPY_DST,
+  });
+  const renderTexture = t.device.createTexture({
+    size: { width: 1, height: 1, depth: 1 },
+    format: 'r8unorm',
+    usage: GPUTextureUsage.COPY_SRC | GPUTextureUsage.OUTPUT_ATTACHMENT,
+  });
+
+  const sampler = t.device.createSampler();
+
+  const bindGroupLayout = t.device.createBindGroupLayout({
+    bindings: [
+      {
+        binding: 0,
+        visibility: GPUShaderStage.FRAGMENT,
+        type: 'sampler',
+      },
+      {
+        binding: 1,
+        visibility: GPUShaderStage.FRAGMENT,
+        type: 'sampled-texture',
+      },
+    ],
+  });
+
+  // fill the sample texture with data
+  const srcBuffer = t.device.createBuffer({
+    size: 4,
+    usage: GPUBufferUsage.COPY_SRC | GPUBufferUsage.COPY_DST,
+  });
+  const data = new Uint32Array([1]);
+  srcBuffer.setSubData(0, data);

```

Comment:
setSubData is not yet* in WebGPU. I'd use `createBufferMappedAsync` instead for now: https://developers.google.com/web/updates/2019/08/get-started-with-gpu-compute-on-the-web#write_buffer_memory

Reply:
Got it, thanks!